jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -393,16 +402,10 @@ private void writeAnnotations(String inPath, String outPath) throws IOException
         inPath.endsWith(".jar") || inPath.endsWith(".class"), "invalid input path - " + inPath);
     LOG(DEBUG, "DEBUG", "Writing Annotations to " + outPath);
 
-    String outFile;
+    new File(outPath).getParentFile().mkdirs();
```

Comment:
It would be better to rename the param to `outFile` instead.

Reply:
Sure, will do this. I have no strong feelings on either name, though.